autoconfigure/pom.xml

```diff
@@ -50,7 +50,7 @@
                     <classifier>module</classifier>
                     <!-- https://github.com/spring-projects/spring-boot/issues/3426 transitive exclude doesn't work -->
                     <excludeGroupIds>
-                        io.zipkin.zipkin2,org.springframework.boot,org.springframework,org.slf4j,com.fasterxml.jackson.core,com.google.auto.value,com.squareup.moshi,com.squareup.okio,com.squareup.okhttp3
+                        io.zipkin.zipkin2,org.springframework.boot,org.springframework,org.slf4j,com.fasterxml.jackson.core
```

Comment:
this line changes what we have to exclude. Since zipkin doesn't package okhttp anymore (and you don't use auto-value), it is invalid to exclude them.